newIDE/app/src/MainFrame/index.js

```diff
@@ -1540,27 +1534,17 @@ const MainFrame = (props: Props) => {
   };
 
   const openCreateProjectDialog = React.useCallback(
-    (tab: CreateProjectDialogTabs) => (open: boolean = true) => {
-      setCreateDialogInitialTab(tab);
+    () => (open: boolean = true) => {
       setState(state => ({ ...state, createDialogOpen: open }));
     },
     [setState]
   );
   const closeCreateDialog = () => {
     setState(state => ({ ...state, createDialogOpen: false }));
   };
-  const onOpenTutorials = React.useMemo(
-    () => openCreateProjectDialog('tutorials'),
-    [openCreateProjectDialog]
-  );
-  const onOpenExamples = React.useMemo(
-    () => openCreateProjectDialog('examples'),
-    [openCreateProjectDialog]
-  );
-  const onOpenGamesShowcase = React.useMemo(
-    () => openCreateProjectDialog('games-showcase'),
-    [openCreateProjectDialog]
-  );
+  const onOpenExamples = React.useMemo(() => openCreateProjectDialog(), [
```

Comment:
This is not needed anymore

Reply:
I actually use it when the user clicks on "create a new project", the list of examples is shown (with the button to create a blank project at the bottom)